protocol_rfcs/catalog-owned.md

```diff
@@ -0,0 +1,489 @@
+
+Delta files are stored as JSON in a directory at the root of the table named `_delta_log`, and
+together with checkpoints make up the log of all changes that have occurred to a table. Delta files
+are the unit of atomicity for a table, and are named using the next available version number,
+zero-padded to 20 digits.
+
+For example:
+
+```
+./_delta_log/00000000000000000000.json
+```
+
+<ins>**Note:** If the [`catalogOwned` table feature](#catalog-owned-tables) is enabled on the table,
+recently ratified commits may not yet be published to the `_delta_log` directory - they may be
+hosted directly by the catalog or reside in the `_delta_log/_staged_commits` directory. Delta
+clients must contact the table's owning catalog in order to find the information about the
+[ratified, potentially-unpublished commits](#publishing-commits).</ins>
+
+<ins>The `_delta_log/_staged_commits` directory is the staging area for proposed [staged](#staged-commit)
+commits. Delta files in this directory have a UUID embedded into them and follow the pattern
+`<version>.<uuid>.json`, where the version corresponds to the proposed commit version, zero-padded
+to 20 digits.</ins>
+
+<ins>For example:</ins>
+
+```
+./_delta_log/_staged_commits/00000000000000000000.3a0d65cd-4056-49b8-937b-95f9e3ee90e5.json
+./_delta_log/_staged_commits/00000000000000000001.7d17ac10-5cc3-401b-bd1a-9c82dd2ea032.json
+./_delta_log/_staged_commits/00000000000000000001.016ae953-37a9-438e-8683-9a9a4a79a395.json
+./_delta_log/_staged_commits/00000000000000000002.3ae45b72-24e1-865a-a211-34987ae02f2a.json
+```
+
+<ins>NOTE: The (proposed) version number of a staged commit is authoritative - file
+`00000000000000000100.<uuid>.json` always corresponds to a commit attempt for version 100. Besides
+simplifying implementations, it also acknowledges the fact that commit files cannot safely be reused
+for multiple commit attempts. For example, resolving conflicts in a table with [row
+tracking](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#row-tracking) enabled requires
+rewriting all file actions to update their `baseRowId` field.</ins>
+
+<ins>The [catalog](#terminology-catalogs) is the source of truth about which staged commit files in
+the `_delta_log/_staged_commits` directory correspond to ratified versions, and Delta clients should
+not attempt to directly interpret the contents of that directory. Refer to
+[catalog-owned tables](#catalog-owned-tables) for more details.</ins>
+
+~~Delta files use new-line delimited JSON format, where every action is stored as a single line JSON
+document. A delta file, `n.json`, contains an atomic set of [_actions_](#actions) that should be
+applied to the previous table state, `n-1.json`, in order to the construct `n`th snapshot of the
+table. An action changes one aspect of the table's state, for example, adding or removing a file.~~
+
+<ins>Delta files use newline-delimited JSON format, where every action is stored as a single-line
```

Comment:
This uses "Delta files", but the preceding text uses "commits". And the section title says "Delta Log Entries". Nothing actually says that a Delta Log Entry == a Delta file == a commit.

It might make sense to move this paragraph above the whole catalogOwned sequence. The first part is really an introduction into what's in these Delta files, and then the catalogOwned is a modification to that. The intermezzo about catalogOwned is really making it hard to pop back up the stack to this higher level intro description.